resources/tutorial/grid2.csv

```diff
@@ -0,0 +1,1002 @@
+"x"
```

Comment:
Do you need this file? If not, please remove it or make it `.local.csv`

Reply:
We used the file to obtain the plots and pushed it by mistake, now we have deleted it.